components/eam/tools/topo_tool/cube_to_target/cube_to_target.F90

```diff
@@ -981,17 +1087,31 @@ end subroutine usage
 !
 !
 !
-subroutine wrtncdf_unstructured(n,terr,sgh,sgh30,lon,lat,fout)
+subroutine wrtncdf_unstructured(n, terr, sgh, sgh30, &
+                                calc_orographic_shape_params, &
+                                nvar_dirOA, nvar_dirOL, &
+                                orographic_convexity, &
```

Comment:
These changes require passing the shape parameter arrays even if we are not calculating/writing them. One alternative might be to add a separate function to write these to the output file, but maybe that's not worth the effort. Just an idea.